api/controller.py

```diff
@@ -1075,6 +1076,34 @@ def search(self, lane_identifier, feed_class=AcquisitionFeed):
             facets=facets
         )
 
+    def qa_feed(self, feed_class=AcquisitionFeed):
+        """Create an OPDS feed containing the information necessary to
+        run a full set of integration tests against this server and
+        the vendors it relies on.
+        """
+
+        library = flask.request.library
+        search_engine = self.search_engine
+        if isinstance(search_engine, ProblemDetail):
+            return search_engine
+
+        url = self.url_for(
+            "qa_feed",
+            library_short_name=library.short_name,
+        )
+
+        jwl = JackpotWorkList(library)
+        annotator = self.manager.annotator(jwl)
+
+        # TODO: Make it possible to pass a pagination object into
+        # groups() to override the standard size of a grouped lane,
+        # improving performance.
+        return feed_class.groups(
```

Comment:
@EdwinGuzman By letting `groups` handle this rather than calling `query_works_multi`, as you did, I've simplified the code a lot at the cost of performance. We can optimize this later by creating some kind of interface that lets a child of a `WorkList` signal to its parent that it can participate in a `query_works_multi` call. Then the parent can make a single `query_works_multi` call and mix in the results with the works it had to get by calling `works` on a bunch of individual `WorkList`s.